code/game/objects/items/storage/backpack.dm

```diff
@@ -398,11 +398,88 @@
 	desc = "A large duffel bag for holding extra things."
 	icon_state = "duffel"
 	inhand_icon_state = "duffel"
-	slowdown = 1
+	actions_types = list(/datum/action/item_action/zipper)
+	/// If this bag is zipped (contents hidden) up or not
+	/// Starts enabled so people need to figure it out to use the thing
+	var/zipped_up = TRUE
 
 /obj/item/storage/backpack/duffelbag/Initialize(mapload)
 	. = ..()
 	atom_storage.max_total_storage = 30
+	set_zipper(TRUE)
+
+/obj/item/storage/backpack/duffelbag/update_desc(updates)
+	. = ..()
+	desc = "[initial(desc)]<br>[zipped_up ? "It's zipped up, can't get in it!" : "It's unzipped, and harder to move in."]"
```

Comment:
Zipped up description sounds a bit weird, like it's intended to be a balloon alert rather than a description
```suggestion
	desc = "[initial(desc)]<br>[zipped_up ? "It's zipped up, preventing you from accessing its contents." : "It's unzipped, and harder to move in."]"
```